Sito/pagamento.html

```diff
@@ -1,26 +1,26 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="it" lang="it">
+<!DOCTYPE html>
```

Comment:
Ho guardato un attimo su internet e leggo che xml:lang non ha senso metterlo in un file html5 quindi sarebbero da togliere tutti gli xml:lang in questa pagina e lasciare solo lang